src/server/cluster/cluster_family.cc

```diff
@@ -522,7 +520,7 @@ void ClusterFamily::DflyClusterConfig(CmdArgList args, ConnectionContext* cntx)
   DCHECK(tl_cluster_config != nullptr);
 
   // TODO rewrite with outgoing migrations
-  if (!StartSlotMigrations(new_config->GetIncomingMigrations(), cntx)) {
+  if (!StartSlotMigrations(new_config->GetOutgoingMigrations(), cntx)) {
```

Comment:
what does this change means? GetIncomingMigrations -> GetOutgoingMigrations
also I think you can remove the todo above

Reply:
the process initiating is moved from the target the target node to the source node so we need to use reverse migration object